back/mail-processing/yarn.lock

```diff
@@ -2964,6 +2964,11 @@ lodash@^4.17.15:
   resolved "https://registry.yarnpkg.com/lodash/-/lodash-4.17.15.tgz#b447f6670a0455bbfeedd11392eff330ea097548"
   integrity sha512-8xOcRHvCjnocdS5cpwXQXVzmmh5e5+saE2QGoeQmbKmRS6J3VQppPOIt0MnmE+4xlZoumy0GPG0D0MVIQbNA1A==
 
+lodash@^4.17.20:
```

Comment:
since only 2 functions used better to use individual packages. e.g. lodash.merge lodash.pick

Reply:
Do you mean for bundle size? I do `import pick from 'lodash/pick'`, or do you menu to reduce node_modules size? I usually use lots of other lodash functions, like debouce, throttle, things like isArray, isObject, uniq, map over objects, etc... Including separate package for each function doesn't really makes sense to me.

Reply:
i mean bundle size.  Idk what for all this functions when there native solutions for most of this things. https://github.com/you-dont-need/You-Dont-Need-Lodash-Underscore

Reply:
If to talk about bundle size, there are so large packages that lodash is barely visible (8kb parsed), probably makes sense to optimize that large things in the first place. About https://github.com/you-dont-need/You-Dont-Need-Lodash-Underscore I don't see deep merge there, generally I can add util functions, but then other devs may add their duplicates, when lodash is used then everyone just knows that it can be used, it has good docs, and it covers most of util cases, with custom utils devs may not know that there is util for this specific case so they put their own or even write utils functions in the place, this can end up with messier code than using lodash as a util lib.